diff -p -up ./otb/bfctl/bfctl.c_orig ./otb/bfctl/bfctl.c
--- ./otb/bfctl/bfctl.c_orig	2009-09-10 10:48:49.000000000 +0200
+++ ./otb/bfctl/bfctl.c	2009-09-10 10:48:49.000000000 +0200
@@ -78,7 +78,7 @@ char			*argv[];
 	ao_setopt(op, "s", "s", 1, AOINT);
 
 	if (ao_parse(op, &argc, argv)) {
-		fprintf(stderr, usage);
+		fprintf(stderr, "%s", usage);
 		exit(errno);
 	}
 
@@ -98,7 +98,7 @@ char			*argv[];
 	}
 
 	if (nid_parse(&argc, argv) || (errno = (argc == 1) ? 0 : EUSAGE)) {
-		fprintf(stderr, usage);
+		fprintf(stderr, "%s", usage);
 		kexit(errno);
 	}
 /*
@@ -107,7 +107,7 @@ char			*argv[];
 	nid_get(&n_index, &nodeid, &n_flags);
 
 	if (n_index < 0) {
-		fprintf(stderr, usage);
+		fprintf(stderr, "%s", usage);
 		kexit(EUSAGE);
 	}
 
diff -p -up ./otb/bfstate/bfstate.c_orig ./otb/bfstate/bfstate.c
--- ./otb/bfstate/bfstate.c_orig	2009-09-10 10:48:49.000000000 +0200
+++ ./otb/bfstate/bfstate.c	2009-09-10 10:48:49.000000000 +0200
@@ -74,7 +74,7 @@ main(int argc, char *argv[])
 	followed("B");
 
 	if (do_args(&argc, argv)) {
-		fprintf(stderr, usage);
+		fprintf(stderr, "%s", usage);
 		exit(errno);
 	}
 
@@ -112,7 +112,7 @@ main(int argc, char *argv[])
 	}
 
 	if (nid_parse(&argc, argv) || (errno = (argc == 1) ? 0 : EUSAGE)) {
-		fprintf(stderr, usage);
+		fprintf(stderr, "%s", usage);
 		kexit(errno);
 	}
 /*
@@ -122,7 +122,7 @@ main(int argc, char *argv[])
 
 	if (n_index < 0) {
 		errno = EUSAGE;
-		fprintf(stderr, usage);
+		fprintf(stderr, "%s", usage);
 		kexit(errno);
 	}
 
diff -p -up ./otb/doom/doom.c_orig ./otb/doom/doom.c
--- ./otb/doom/doom.c_orig	2009-09-10 10:48:49.000000000 +0200
+++ ./otb/doom/doom.c	2009-09-10 10:48:49.000000000 +0200
@@ -77,7 +77,7 @@ main(int argc, char *argv[])
 	followed("u");
 
 	if (do_args(&argc, argv)) {
-		fprintf(stderr, usage);
+		fprintf(stderr, "%s", usage);
 		exit(errno);
 	}
 /*
@@ -92,12 +92,12 @@ main(int argc, char *argv[])
  */
 	if (opt_taken('0')) {
 		if (opt_taken('u') || intparam('0', &signo) || signo <= 0) {
-			fprintf(stderr, usage);
+			fprintf(stderr, "%s", usage);
 			exit(EUSAGE);
 		}
 	} else if (opt_taken('u')) {
 		if (opt_taken('k') || intparam('u', &signo) || signo < 0) {
-			fprintf(stderr, usage);
+			fprintf(stderr, "%s", usage);
 			exit(EUSAGE);
 		} else {
 			signo = -signo;
@@ -118,7 +118,7 @@ main(int argc, char *argv[])
 
 	if ((nid_parse(&argc, argv)) || (pid_parse(&argc, argv)) ||
 			(errno = (argc == 1) ? 0 : EUSAGE)) {
-		fprintf(stderr, usage);
+		fprintf(stderr, "%s", usage);
 		kexit(errno);
 	}
 /*
diff -p -up ./otb/fctl/fctl.c_orig ./otb/fctl/fctl.c
--- ./otb/fctl/fctl.c_orig	2009-09-10 10:48:49.000000000 +0200
+++ ./otb/fctl/fctl.c	2009-09-10 10:48:49.000000000 +0200
@@ -75,7 +75,7 @@ main(int argc, char *argv[])
 	exclusive("Ss");
 
 	if (do_args(&argc, argv)) {
-		fprintf(stderr, usage);
+		fprintf(stderr, "%s", usage);
 		exit(errno);
 	}
 /*
@@ -97,7 +97,7 @@ main(int argc, char *argv[])
 	}
 
 	if ((nid_parse(&argc, argv)) || (errno = (argc <= 2) ? 0 : EUSAGE)) {
-		fprintf(stderr, usage);
+		fprintf(stderr, "%s", usage);
 		kexit(errno);
 	}
 /*
@@ -106,7 +106,7 @@ main(int argc, char *argv[])
 	nid_get(&n_index, &nodeid, &n_flags);
 
 	if (n_index < 0) {
-		fprintf(stderr, usage);
+		fprintf(stderr, "%s", usage);
 		kexit(EUSAGE);
 	}
 
diff -p -up ./otb/filedr/filedr.c_orig ./otb/filedr/filedr.c
--- ./otb/filedr/filedr.c_orig	2009-09-10 10:48:49.000000000 +0200
+++ ./otb/filedr/filedr.c	2009-09-10 10:48:49.000000000 +0200
@@ -91,7 +91,7 @@ char			*argv[];
 	twoletter("wor");
 
 	if (do_args(&argc, argv) || (errno = (argc == 1) ? 0 : EUSAGE)) {
-		fprintf(stderr, usage);
+		fprintf(stderr, "%s", usage);
 		exit(errno);
 	}
 /*
@@ -357,7 +357,7 @@ fdr_init()
 			fl_wrrd = TRUE;
 			flags = flags | LAM_O_RDWR;
 		} else {
-			fprintf(stderr, usage);
+			fprintf(stderr, "%s", usage);
 			kexit(EUSAGE);
 		}
 	}
@@ -397,7 +397,7 @@ fdr_init()
 		else if (strchr(str_flags,'e') != NULL)
 				flags = flags | LAM_O_REUSE;
 		else {
-			fprintf(stderr, usage);
+			fprintf(stderr, "%s", usage);
 			kexit(EUSAGE);
 		}
 	}
diff -p -up ./otb/fstate/fstate.c_orig ./otb/fstate/fstate.c
--- ./otb/fstate/fstate.c_orig	2009-09-10 10:48:49.000000000 +0200
+++ ./otb/fstate/fstate.c	2009-09-10 10:48:49.000000000 +0200
@@ -76,7 +76,7 @@ main(int argc, char *argv[])
 	followed("F");
 
 	if (do_args(&argc, argv)) {
-		fprintf(stderr, usage);
+		fprintf(stderr, "%s", usage);
 		exit(errno);
 	}
 /*
@@ -110,7 +110,7 @@ main(int argc, char *argv[])
 	}
 
 	if (nid_parse(&argc, argv) || (errno = (argc == 1) ? 0 : EUSAGE)) {
-		fprintf(stderr, usage);
+		fprintf(stderr, "%s", usage);
 		kexit(errno);
 	}
 /*
@@ -129,7 +129,7 @@ main(int argc, char *argv[])
 	}
 
 	else {
-		fprintf(stderr, usage);
+		fprintf(stderr, "%s", usage);
 		kexit(errno);
 	}
 
diff -p -up ./otb/ipcdr/ipcdr.c_orig ./otb/ipcdr/ipcdr.c
--- ./otb/ipcdr/ipcdr.c_orig	2009-09-10 10:48:49.000000000 +0200
+++ ./otb/ipcdr/ipcdr.c	2009-09-10 10:48:49.000000000 +0200
@@ -89,7 +89,7 @@ char			*argv[];
 	exclusive("CDFI");
 
 	if (do_args(&argc, argv)) {
-		fprintf(stderr, usage);
+		fprintf(stderr, "%s", usage);
 		exit(errno);
 	}
 /*
@@ -111,7 +111,7 @@ char			*argv[];
 	}
 
 	if (nid_parse(&argc, argv) || (errno = (argc == 1) ? 0 : EUSAGE)) {
-		fprintf(stderr, usage);
+		fprintf(stderr, "%s", usage);
 		kexit(errno);
 	}
 /*
@@ -166,7 +166,7 @@ sender()
 	nid_get(&n_index, &nodeid, &n_flags);
 
 	if (n_index < 0) {
-		fprintf(stderr, usage);
+		fprintf(stderr, "%s", usage);
 		kexit(EUSAGE);
 	}
 
diff -p -up ./otb/lamexec/lamexec.c_orig ./otb/lamexec/lamexec.c
--- ./otb/lamexec/lamexec.c_orig	2009-09-10 10:48:49.000000000 +0200
+++ ./otb/lamexec/lamexec.c	2009-09-10 10:48:49.000000000 +0200
@@ -121,7 +121,7 @@ main(int argc, char **argv)
     }
 
     if (ao_parse(ad, &argc, argv))  {
-	fprintf(stderr, usage);
+	fprintf(stderr, "%s", usage);
 	exit(errno);
     }
 /*
@@ -243,7 +243,7 @@ build_job(int argc, char **argv, char **
 	char		*qarg;			/* quoted argument */
 
 	if (nid_parse(&argc, argv) || (errno = (argc > 1) ? 0 : EUSAGE)) {
-		fprintf(stderr, usage);
+		fprintf(stderr, "%s", usage);
 		return(0);
 	}
 /*
@@ -256,7 +256,7 @@ build_job(int argc, char **argv, char **
  * Parse the app. schema.
  */
 		if ((argc != 2) || ao_taken(ad, "s")) {
-			fprintf(stderr, usage);
+			fprintf(stderr, "%s", usage);
 			kexit(EUSAGE);
 		}
 
diff -p -up ./otb/loadgo/loadgo.c_orig ./otb/loadgo/loadgo.c
--- ./otb/loadgo/loadgo.c_orig	2009-09-10 10:48:49.000000000 +0200
+++ ./otb/loadgo/loadgo.c	2009-09-10 10:48:49.000000000 +0200
@@ -97,7 +97,7 @@ main(int argc, char **argv)
     followed("s");
 
     if (do_args(&argc, argv)) {
-	fprintf(stderr, usage);
+	fprintf(stderr, "%s", usage);
 	exit(errno);
     }
 /*
@@ -123,7 +123,7 @@ main(int argc, char **argv)
     }
 
     if (nid_parse(&argc, argv)) {
-	fprintf(stderr, usage);
+	fprintf(stderr, "%s", usage);
 	kexit(errno);
     }
 
@@ -132,7 +132,7 @@ main(int argc, char **argv)
     --argc;
 
     if ((index < 0) || (argc == 0)) {
-	fprintf(stderr, usage);
+	fprintf(stderr, "%s", usage);
 	kexit(EUSAGE);
     }
 /*
diff -p -up ./otb/state/state.c_orig ./otb/state/state.c
--- ./otb/state/state.c_orig	2009-09-10 10:48:49.000000000 +0200
+++ ./otb/state/state.c	2009-09-10 10:48:49.000000000 +0200
@@ -86,7 +86,7 @@ char   			*argv[];
 	validopts("ghklt");
 
 	if (do_args(&argc, argv)) {
-		fprintf(stderr, usage);
+		fprintf(stderr, "%s", usage);
 		exit(errno);
 	}
 
@@ -104,7 +104,7 @@ char   			*argv[];
 
 	if (nid_parse(&argc, argv) || (pid_parse(&argc, argv)) ||
 			(errno = (argc == 1) ? 0 : EUSAGE)) {
-		fprintf(stderr, usage);
+		fprintf(stderr, "%s", usage);
 		kexit(errno);
 	}
 /*
@@ -113,7 +113,7 @@ char   			*argv[];
 	nid_get(&n_index, &nodeid, &n_flags);
 
 	if (n_index < 0) {
-		fprintf(stderr, usage);
+		fprintf(stderr, "%s", usage);
 		kexit(errno);
 	}
 
